server/game/gamesteps/taxation/discardtoreserveprompt.js

```diff
@@ -38,7 +38,7 @@ class DiscardToReservePrompt extends BaseStep {
         // on the top of the discard pile.
         cards = cards.reverse();
         player.discardCards(cards, false, () => {
-            this.game.addMessage('{0} discards {1}', player, cards);
+            this.game.addMessage('{0} discards {1} to match reserve', player, cards);
```

Comment:
As a minor nitpick i'd say that it probably makes more sense to say 'meet reserve' here

Reply:
That's better. 👍